modules/oadp-creating-backup-cr.adoc

```diff
@@ -52,11 +52,15 @@ spec:
   hooks: {}
   includedNamespaces:
   - <namespace> <1>
-  storageLocation: <velero-sample-1> <2>
+  includedResources: [] <2>
+  excludedResources: [] <3>
+  storageLocation: <velero-sample-1> <4>
   ttl: 720h0m0s
 ----
 <1> Specify an array of namespaces to back up.
-<2> Specify the name of the `backupStorageLocations` CR.
+<2> Optional. Specify an array of resources to include in the backup. Resources may be shortcuts (for example 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
```

Comment:
Please add a comma after "for example" (throughout).

Reply:
@RichardHoch 
Done